### ISSUE TITLE
Support for the Smart Plug Feature of the RE270K and RE370K WiFi Extenders

### ISSUE THREAD

**Issue by philshare** [DaveGut/TP-Link-SmartThings#10]
I am just starting out with SmartThings and GitHub so I apologize if I am overreaching or asking a question that is against the rules.

The RES270K and the RE370K WiFi extenders physically resemble HS100 smart plugs but with a WiFi antenna. The smart plug configures in Kasa like other similar devices. What would it take to be able to control the smart plug aspect in SmartThings like we do other TP-Link smart plugs.=?

Would I blow everything up for myself if I just added them to the Cloud Connect Plug-Switch Devices (no energy monitor capability) list?

Thanks,

Phil

**Comment by DaveGut** [comment]
Did you resolve this?  If not, please post on the SmartThings topic.  I need your SmartApp details page value for state.devices (it will tell me the device names to add to the list in the SmartApp).

Dave

**Comment by philshare** [comment]
Hi Dave,

Thanks for getting back to me. I'm very new to SmartThings and also GitHUB so I have been proceeding very slowly. I haven't resolved this because, honestly, there is too much I'm not sure about.

The SmartThings topic I created is at:
https://community.smartthings.com/t/add-support-for-re270k-and-re370k-to-tp-link-cloud-connect-service-manager-and-tp-link-device-handlers/152126

I hope that's OK.

I need you to help me with how to obtain my SmartApp details page value for you. I looked around and couldn't see anything that might answer your requirement. In terms of SmartApp, I believe the only custom one I'm using is yours.
